cmd/migrate_from_qdrant.go

```diff
@@ -624,6 +624,8 @@ func (r *MigrateFromQdrantCmd) getOffsetPoint(ctx context.Context, sourceClient
 	return points[0], nil
 }
 
-func (r *MigrateFromQdrantCmd) getOffsetPointId() *qdrant.PointId {
-	return qdrant.NewIDNum(1)
+func (r *MigrateFromQdrantCmd) getOffsetPointId(sourceCollection string) *qdrant.PointId {
+	deterministicUUID := uuid.NewSHA1(uuid.NameSpaceURL, []byte(sourceCollection))
+
+	return qdrant.NewIDUUID(deterministicUUID.String())
```

Comment:
Generate a deterministic point ID based on the source collection name.

Can also be based on source host + source collection name if needed.